src/main/java/org/dwcj/interfaces/Visible.java

```diff
@@ -1,6 +1,11 @@
 package org.dwcj.interfaces;
 
-public interface IVisible {
+
+/**
+ * Interface facilitates implementation of behaviors to modify a 
+ * controls visibility on a page
+ */
+public interface Visible {
```

Comment:
I'm not sure about this. but do you think `HasVisible` is better ?

Reply:
I thought about HasVisible but that sounded a little off to me for some reason - what about HasVisibility?

Reply:
`HasVisibility` sounds good. Let's go with that, it goes with the pattern we used for other interfaces.